### ISSUE TITLE
Problem with LIKQ build?

### ISSUE THREAD

**Issue by MasakaGray** [microsoft/GraphEngine#325]
I am new to this and a bit struggled to build it. Currently, I could build Trinity.Core.dll in 2019.
However, I can't build likq. The error list is long and looks like all staff in TSL files are not included. 
Really appreciated if anyone could help me to build it.

**Comment by TaviTruman** [comment]
@XiaAlexFang I also ran into a number of build errors myself; please take a look at #324 You may need to clear the Nuget Cache and try to build again.

**Comment by TaviTruman** [comment]
Please take a screenshot of the build errors so that we can help you out.

**Comment by MasakaGray** [comment]
<img width="1440" alt="Screenshot 2020-06-17 at 09 10 32" src="https://user-images.githubusercontent.com/26504936/84872979-785bc400-b07a-11ea-8e80-02781d8fc859.png">

<img width="1440" alt="Screenshot 2020-06-17 at 09 11 18" src="https://user-images.githubusercontent.com/26504936/84873062-91fd0b80-b07a-11ea-8b37-6685fa9e88ea.png">

> Please take a screenshot of the build errors so that we can help you out.

Thanks for the quick reply. Please see the following pics. It looks like that the tsl files are not connected. I tried the sample example Friends as well. I didn't not use the vs extension. I build the trinity core and links it to the Friend, as you can see it in the csproj file.

**Comment by MasakaGray** [comment]
Previous I use vs extension for 2015 then I could build these examples. However, since I failed to build vs extension for 2019, so I tried to link trinity core directly, but I am not sure that I am doing the right way. And also how to solve the building failure of vs extension 2019?

I know I asked a lot of questions...but I am new to it and really struggled learning how to use it. Thanks a lot !

<img width="1440" alt="Screenshot 2020-06-17 at 09 38 58" src="https://user-images.githubusercontent.com/26504936/84875847-6e3bc480-b07e-11ea-9f21-e5a48517769f.png">
